ccmlib/node.py

```diff
@@ -739,12 +730,9 @@ def wait_for_compactions(self):
         """
         Wait for all compactions to finish on this node.
         """
-        pattern = re.compile("pending tasks: 0")
-        while True:
-            output, err = self.nodetool("compactionstats", capture_output=True)
-            if pattern.search(output):
-                break
-            time.sleep(10)
+        timeout = 24 * 60 * 60  # Wait 24 hours
+        wait_for(func=lambda: re.compile("pending tasks: 0").search(self.nodetool(
```

Comment:
@nyh 
I can't remove the `timeout` variable because it's required parameter in `wait_for` method